packages/frontend-v2/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@graduate/frontend-v2",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@graduate/common": "workspace:^",
```

Comment:
use `workspace:*`, see https://yarnpkg.com/features/workspaces/#workspace-ranges-workspace

Reply:
ooh nice thank you for catching this!!